2-mandatory/2-water-bottle.js

```diff
@@ -19,18 +19,27 @@ let bottle = {
   volume: 0,
   fillUp: function () {
     // calling this function should completely fill your bottle (volume = 100);
+    this.volume = 100
   },
   pour: function () {
     // calling this function should increase your bottle volume by 10 units;
+   if (this.volume < 100 ) {
+    return this.volume += 10 ;
+   }
   },
   drink: function () {
     // calling this function should decrease your bottle volume by 10 units;
+    if(this.volume > 0){
+      return this.volume -= 10;
+    }
```

Comment:
Similar to the previous one, great start but consider if the value was 7 and you subtracted 10 from it